chalicelib/check_setup.json

```diff
@@ -1,438 +1,516 @@
-        "group": "Elasticsearch checks",
-        "conditions": ["put_env"],
+        "group": "Elasticsearch Checks",
+        "conditions": [
+            "put_env"
+        ],
         "schedule": {
-            "thirty_min_checks": {
-                "all": {
-                    "kwargs": {"primary": true},
-                    "dependencies": []
+            "manual_checks": {
+                "<env-name>": {
+                    "kwargs": {
+                        "primary": true
+                    }
                 }
             }
-        }
+        },
+        "display": [
+            "<env-name>"
+        ]
     },
     "indexing_records": {
         "title": "Indexing records",
-        "group": "Elasticsearch checks",
-        "conditions": ["put_env"],
+        "group": "Elasticsearch Checks",
+        "conditions": [
+            "put_env"
+        ],
         "schedule": {
-            "thirty_min_checks": {
-                "all": {
-                    "kwargs": {"primary": true},
-                    "dependencies": []
+            "manual_checks": {
+                "<env-name>": {
+                    "kwargs": {
+                        "primary": true
+                    }
                 }
             }
-        }
+        },
+        "display": [
+            "<env-name>"
+        ]
     },
     "change_in_item_counts": {
         "title": "Change in item counts",
-        "group": "Metadata checks",
+        "group": "Metadata Checks",
         "schedule": {
-            "morning_checks_2": {
-                "all": {
-                    "kwargs": {"primary": true},
-                    "dependencies": []
+            "manual_checks": {
+                "<env-name>": {
+                    "kwargs": {
+                        "primary": true
+                    }
                 }
             }
-        }
+        },
+        "display": [
+            "<env-name>"
+        ]
     },
     "page_children_routes": {
         "title": "Pages with bad routes",
-        "group": "Audit checks",
+        "group": "Audit Checks",
         "schedule": {
-            "morning_checks": {
-                "all": {
-                    "kwargs": {"primary": true},
-                    "dependencies": []
+            "manual_checks": {
+                "<env-name>": {
+                    "kwargs": {
+                        "primary": true
+                    }
                 }
             }
-        }
+        },
+        "display": [
+            "<env-name>"
+        ]
     },
     "workflow_properties": {
         "title": "Workflows with missing or duplicate steps properties",
-        "group": "Audit checks",
+        "group": "Audit Checks",
         "schedule": {
-            "morning_checks": {
-                "all": {
-                    "kwargs": {"primary": true},
-                    "dependencies": []
+            "manual_checks": {
+                "<env-name>": {
```

Comment:
This is another where we might want a less frequent schedule, and/or add a param so that by default it only looks at workflows last modified in the past week.